app/assets/javascripts/analytics.js

```diff
@@ -15,9 +15,27 @@ if (document.head.dataset.piwikHost) {
     e = e || window.event;
     var target = e.target || e.srcElement;
 
-    // Track if a user clicked on the verify button for a Wordpress flow
+    // A switch would feel better, but would eval on target.dataset actually work?
```

Comment:
yes I think a switch would be more legit

Reply:
another way you could do this is just to bind more data attributes, like one for `WordpressVerificationClicked`, one for `Clicked`, one for `WordpressFlow`. Then for all elements which have these data attributes, send the data the element contains.

Reply:
other nit is please don't use comments for code questions, in case we merge the code the funny comments will get stuck. it's better to leave a Github comment on the PR .

if u must write FIXME or XXX or HACK in the comment